entities/regions.py

```diff
@@ -0,0 +1,42 @@
+from random import randint
+from random import choice
+from config import Config
+
+
+class Regions(object):
+
+    def __init__(self, region_id, subregion_id):
+        self.region_id = region_id
+        self.subregion_id = subregion_id
+
+    def region(self):
+        global region
```

Comment:
Don't use globals when you don't need to (you almost never need to - it's purely by (bad) design that you end up in a situation where you do).

This is not one of those cases.